Xamarin.PropertyEditing.Tests/MockControls/MockSampleControl.cs

```diff
@@ -52,9 +52,9 @@ public enum Enumeration
 		[Flags]
 		public enum Flags
 		{
-			FlagOne,
-			FlagTwo,
-			FlagThree
+			FlagOne = 1,
```

Comment:
If we don't add values to the Flagged Enum, it treats the first Enum as 0, which then messes up all toggling logic and therefore doesn't work correctly.

Reply:
No, it's supposed to be able to handle a 0, leave this as is.